src/library/Uno.Material/Styles/Controls/v2/CheckBox.xaml

```diff
@@ -17,44 +17,178 @@
 	<x:Double x:Key="CheckAreaSize">18</x:Double>
 	<CornerRadius x:Key="CheckAreaCornerRadius">4</CornerRadius>
```

Comment:
Should it be better for these resources to start the name with `CheckBox`... ?
Similar to https://github.com/unoplatform/uno/blob/3dc75d23a3e4635e32432aab4bd4ffccc023a249/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/CheckBox_themeresources.xaml#L364

Reply:
Yeah maybe we should prefix it with MaterialCheckBox